lib/galaxy/authnz/psa_authnz.py

```diff
@@ -357,12 +357,12 @@ def disconnect(name=None, user=None, user_storage=None, strategy=None,
     Additionally, returning any value except for a(n) (empty) dictionary, will break the
     disconnect pipeline, and that value will be returned as a result of calling the `do_disconnect` function.
     """
-    user_authnz = strategy.trans.sa_session.query(user_storage).filter(user_storage.table.c.user_id == user.id,
+    user_authnz = user_storage.sa_session.query(user_storage).filter(user_storage.table.c.user_id == user.id,
```

Comment:
I'd assign `user_storage.sa_session` to a variable (e.g. `sa_session`) and use that in the 3 occurrences.